lib/main_common.pm

```diff
@@ -1964,7 +1973,11 @@ sub load_iso_in_external_tests {
 sub load_x11_installation {
     set_var('NOAUTOLOGIN', 1) if is_opensuse;
     load_boot_tests();
-    load_inst_tests();
+    if (is_agama_installer) {
+        load_agama_inst_tests();
+    } else {
+        load_inst_tests();
+    }
```

Comment:
So, because of the following is never executed (basically live tests aren't considered default schedule):

Removing livetest from the test settings should be good enough

https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/f9b63f416995db83a4d8be70154c10579c899d7d/products/opensuse/main.pm#L334C18-L334C26

```suggestion
    if (is_agama_installer) {
        load_agama_inst_tests();
    } else {
        load_inst_tests();
    }
```

Reply:
If after changing the livetest parameter still fails, change this (or add a specific one) to `installation/bootloader_start`

https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/f9b63f416995db83a4d8be70154c10579c899d7d/lib/main_common.pm#L419C32-L419C42